Release Notes/2018/2018-03-27-cloud-platform-release-notes.md

```diff
@@ -0,0 +1,45 @@
+{{{
+"title": "Cloud Platform - Release Notes: March 27, 2018",
+"date": "03-27-2018",
+"author": "Jason Oldham",
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+### New Features (1)
+
+#### [Cloud Application Manager](//www.ctl.io/cloud-application-manager/)
+
+##### Support for Azure Canada
+
+Cloud Application Manager now supports the creation of Azure reseller accounts into Canada territory, so any Canadian customer can now create an Azure account optimized by CenturyLink for consolidated billing and platform support.
+
+### Enhancements (1)
+
+#### Cloud Application Manager
+
+##### [Application Lifecycle Management](//www.ctl.io/cloud-application-manager/application-lifecycle-management/)
+
+The network area of a VM policy for Google Compute Engine has been enhanced to support the subnetworks concepts in Google. Now you are able to deploy instances in Google subnets.
+
+### Announcements (2)
+
+#### [Dedicated Cloud Compute (DCC)](//www.ctl.io/dedicated-cloud-compute/)
+
+##### OS Availability
+
+Due to Microsoft Support End of Life, customers can no longer Clone or Save Image for Windows Server 2003 VMs (guests) in SavvisStation Portal.
+
+#### [Load Balancer as a Service (LBaaS)](//www.ctl.io/load-balancing/)
+
+CenturyLink Cloud has released a new load balancer service in our DE1, IL1 and NY1 locations. We’re calling it Load Balancer as a Service (LBaaS). We added more features that allow you to focus on your business while we help service your infrastructure. You get to keep all of the features that are available in the current Shared Load Balancer service, and we’ve added:
+
+* TCP load balancing
+* Load balance on any port 23 and up
+* Configurable health checks
+
+A single instance hour of the LBaaS service will be priced at $0.025. In addition, each GB of data processed by the load balancing service will be priced at $0.008.
```

Comment:
we should not put pricing in kbs directly. always point to ctl.io/pricing for costs as any change in pricing requires someone to go back and find these one off figures later which is wasteful of time.